docs/snippets/all/concepts/viscomp-visualizer-override-multiple.py

```diff
@@ -1,26 +0,0 @@
-"""Override a visualizer."""
```

Comment:
removal of those snippets belongs to a different PR that also removes their usage in the docs pages I reckon?